frontend/catalyst/device/qjit_device.py

```diff
@@ -223,6 +232,9 @@ def get_qjit_device_capabilities(target_capabilities: DeviceCapabilities) -> Dev
     qjit_capabilities.observables = intersect_operations(
         target_capabilities.observables, RUNTIME_OBSERVABLES
     )
+    qjit_capabilities.measurement_processes = intersect_mps(
+        target_capabilities.measurement_processes, RUNTIME_MPS
```

Comment:
Nice thank you for adding, it. If this variable didn't exist before, do we need to update the verification/decomposition code somewhere to be sure it fetches it from here?

Reply:
Ah good point, never mind 😅